lib/index.js

```diff
@@ -167,6 +167,12 @@ class PackageJson {
     return this
   }
 
+  fromContent (data) {
```

Comment:
I think this method was ill conceived.  In my head it *was* the implementation that this PR is now providing.

Having it public was probably a mistake, as it should probably also have `#canSave` set to false if used directly (and `load` should not use it directly).

That's neither here nor there for this PR but those are the thoughts that this PR makes me have.

Reply:
`fromComment` seems to have the same issue.  I think that for how we use this it's fine, but some of these public methods could be used in a way that gives weird behavior when folks try to save.